### PR TITLE
did a landing page(home)

### PR DESCRIPTION
**Pull request by Bathande**
did landing page(Home)

### CODE REVIEW

File: skietbaan_fe/src/components/List.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          <td><b>USERNAME</b></td>
+          <td><b>EMAIL</b></td>
+          </tr>
+        </thead>
+        <tbody>
+          {this.state.posts.map(post => (
+            <tr key={post.id}>
+            <td><h5>{post.id}</h5></td>
+              <td><h5>{post.username}</h5></td>
+              <td>{post.email}</td>
```
Comment (BWMuller) [inline skietbaan_fe/src/components/List.js]: why are you listing users emails and usernames here? Does this no violate user privacy. Also why does server even return that. Please also ensure that, the user object that is returned does not contain the password hash as well

File: skietbaan_fe/src/components/List.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        <thead>
+         <tr>
+          <td><b>USER_ID</b></td>
+          <td><b>USERNAME</b></td>
+          <td><b>EMAIL</b></td>
+          </tr>
+        </thead>
+        <tbody>
+          {this.state.posts.map(post => (
+            <tr key={post.id}>
```
Comment (laurent-mbuyu) [inline skietbaan_fe/src/components/List.js]: Knowing that the key won't be computed, you can make it a string.

File: skietbaan_fe/src/components/NavbarMenu.js [hunk 3]
```diff
@@ -17,7 +16,7 @@ import { Navbar,
   <Navbar.Collapse id="basic-navbar-nav">
     <Nav className="mr-auto">
       <Nav.Link href="/home">Leader Board</Nav.Link>
-      <Nav.Link href="/login">Score Capture</Nav.Link>
+      <Nav.Link href="/Scores">Score Capture</Nav.Link>
```
Comment (laurent-mbuyu) [inline skietbaan_fe/src/components/NavbarMenu.js]: Consistency on your naming style. Change the Score to score

File: skietbaan_fe/src/components/NavbarMenu.js [hunk 4]
```diff
@@ -17,7 +16,7 @@ import { Navbar,
...
       <Nav.Link href="/login">Notifications</Nav.Link>
```
Comment (laurent-mbuyu) [inline skietbaan_fe/src/components/NavbarMenu.js]: Notifications and documents redirect you to the same location.
Is it intentional?

File: skietbaan_fe/src/components/List.js (hunk 1 above)
Comment (Bathande) [reply skietbaan_fe/src/components/List.js]: this was for testing if i retrieve the data  it will change once we have real data in database

Comment (BWMuller) [reply skietbaan_fe/src/components/List.js]: If its for testing why is it being committed?

Comment (Bathande) [reply skietbaan_fe/src/components/List.js]: i have changed it

File: skietbaan_fe/src/App.js.orig [hunk 5]
```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import './App.scss';
+import { Provider } from 'react-redux';
+import store from './store';
+import { getUser } from './api/github';
+import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
+import Login from './components/Login';
+import NavbarMenu from './components/NavbarMenu';
+import RegisterMember from './components/RegisterMember';
+<<<<<<< HEAD
```
Comment (BWMuller) [inline skietbaan_fe/src/App.js.orig]: Merge remnants. Please don't commit merge headers

File: skietbaan_fe/src/App.js [hunk 6]
```diff
@@ -7,9 +7,10 @@ import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
 import Login from './components/Login';
 import NavbarMenu from './components/NavbarMenu';
 import RegisterMember from './components/RegisterMember';
+import List from './components/List';
 import Landing from './components/landing';
 import ScoreCapture from './components/ScoreCapture';
-class App extends Component {
+class App exteds Component {
```
Comment (clewarne) [inline skietbaan_fe/src/App.js]: should be `extends` Does your code build?

File: skietbaan_fe/src/App.js.orig [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+import NavbarMenu from './components/NavbarMenu';
+import RegisterMember from './components/RegisterMember';
+
+import List from './components/List';
+import Landing from './components/landing';
+
+class App extends Component {
+
+    componentDidMount () {
+        getUser('vnglst').then(data => {
```
Comment (clewarne) [inline skietbaan_fe/src/App.js.orig]: what is this magic string `vnglst` for? Maybe pull out into a constant

File: skietbaan_fe/src/components/Groups.js [hunk 8]
```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react'
+
+ class Groups extends Component {
+     constructor(props){
+         super(props);
+         posts:[]
+     }
+     
+     componentWillMount() {
+        fetch('http://skietbaan.retrotest.co.za/api/User')
```
Comment (clewarne) [inline skietbaan_fe/src/components/Groups.js]: Are you not using React Redux? Don't do http calls in `componentWillMount`. Use the React-Redux pattern and send the data through the redux state.

File: skietbaan_fe/src/components/List.js [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+import '../components/ListStyle.css'
+class List extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      posts :[]
+    }
+  }
+  componentWillMount() {
+    fetch('http://skietbaan.retrotest.co.za/api/User')
```
Comment (clewarne) [inline skietbaan_fe/src/components/List.js]: Seeing this fetch copy pasted multiple times.... pull into a method you can import wherever you need to use it.

**Pull request merged**